incubator/kube-backup/templates/mysql.cronjob.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.mysql.enabled -}}
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: {{ template "fullname_mysql" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  concurrencyPolicy: {{ .Values.mysql.concurrencyPolicy }}
+#  successfulJobsHistoryLimit: {{ .Values.mysql.successfulJobsHistoryLimit }}
```

Comment:
Current helm version does not support that options. Failed on lint.
Commented until helm will support k8s 1.6 ( should be in next release )